src/stdatamodels/jwst/datamodels/schemas/conv_kernel.schema.yaml

```
@@ -0,0 +1,16 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://stsci.edu/schemas/jwst_datamodel/conv_kernel.schema"
+allOf:
+- $ref: referencefile.schema
+- type: object
+  patternProperties:
+    "nrc[ab][1-4]":
```

```suggestion
    "^(nrca1|nrca2|nrca3|nrca4|nrcalong|nrcb1|nrcb2|nrcb3|nrcb4|nrcblong|nrs1|nrs2|mirimage|mirifulong|mirifushort|nis|guider1|guider2)$":
```
As discussed with @tapastro and inspired by the detector list in `keyword_detector`:
https://github.com/spacetelescope/stdatamodels/blob/4ea7d9134cbe1f4a9fc554ecf0d4ba99ba811684/src/stdatamodels/jwst/datamodels/schemas/keyword_pdetector.schema.yaml#L17-L19
the suggested regex here aims to match all lowercase detector names.

---

@penaguerrero Does is make sense to include `guider1` and `guider2` here or is this for a step that will never process guider data?